### DISCUSSION TITLE
Hide Lists section

### DISCUSSION THREAD

**Discussion by rabbitinspace** [community/community#63110] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

When I open my starred repos, the first thing I'm seeing is this huge and useless prompt:
<img width="922" alt="Screenshot 2023-08-08 at 5 20 13 pm" src="https://github.com/community/community/assets/31989081/802af42c-7445-45d6-998d-7e4d3b7730b5">

I don't want to create any lists, I want to browse starred repos without distraction, and this thing eats a lot of vertical space.

**Comment by KAWATKONGCALEUM** [comment]
[]()

**Reply by Cesar556** [reply]
Ok gracias

**Comment by emporteme** [comment]
It seems like you're describing a user experience issue related to a web page or application. However, your description is cut off and I don't have access to external content or visuals. Could you please provide more context or details about the prompt you're seeing and what you consider to be the issue with it? This will help me better understand the problem and provide you with appropriate suggestions or solutions.

**Reply by Cesar556** [reply]
Ok

**Comment by rabbitinspace** [comment]
Hi,

I'm opening github.com, then after logging into my account I navigate to the "Your stars" page. This is how that page is looking for me:

<img width="1278" alt="Screenshot 2023-08-10 at 12 13 17 pm" src="https://github.com/community/community/assets/31989081/ca24cbc6-04f8-4062-85e4-4643bd3833be">

As you can see, the new Lists section is empty and there's no way to hide it. It moves my starred repositories (which I expect to see when I open "Your stars" page) is now on the bottom half of the page which confuses me for a second every time I open the page.

I do not want to create any lists given the small number of stared repos I have, and I do not plan to create them in the future. I want to hide this new section so it doesn't confuse me, and so I can actually see my starred repos when I open "Your stars" page.

**Reply by Cesar556** [reply]
- gracias estoy en eso
